### DISCUSSION TITLE
FR: Add HTML partial for Appendix.

### DISCUSSION THREAD

**Discussion by batpigandme** [quarto-dev/quarto-cli#8632] [Feature Requests]
### Description

A bit related to #6552 in terms of wanting more cutomization, but a bit more minimalistic (i.e. would still be an html partial): Would it be possible to have an HTML template partial for the Appendix (the current `<div id="quarto-appendix">`)?

I find myself wanting to make slight changes to that section, similar to the kind of tweaks I've made with the [`title-block.html`](https://github.com/quarto-dev/quarto-cli/blob/main/src/resources/formats/html/pandoc/title-block.html) partial, but, since the appendix is in `$body$`, I don't have easy access to the variables used. 

For example, I'd like to employ the wording from distill's [reuse-note](https://github.com/rstudio/distill/blob/d88adcaacd6b3b4e96e3f2b88449b4b839ddff0a/R/appendices.R#L89-L111), but I think I'd have to make an entirely new template in order to do so right now.

Thanks for your consideration, and/or any workarounds that currently exist but I haven't figured out!

**Comment by cscheid** [comment]
That makes total sense to me, at least. I'll create a ticket.

**Reply by mcanouil** [reply]
Closing as outdated and in favour of:
- https://github.com/quarto-dev/quarto-cli/issues/8636
